controllers/istio_controller_test.go

```diff
@@ -78,10 +105,7 @@ var _ = Describe("IstioController", Ordered, func() {
 				},
 				Spec: v1.IstioSpec{
 					Version: istioVersion,
-					Values: []byte(`{
-						"pilot":{"image":"` + pilotImage + `"},
-						"istio_cni":{"enabled":true}
-					}`),
+					Values:  istioValues,
```

Comment:
Not sure if it's a good idea to change the values of the existing test. Better to add a new test.